src/App.jsx

```diff
@@ -14,33 +15,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const FIELD_ALPHABETIC = 'alphabetically';
+const FIELD_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [sortBy, setSortBy] = useState(null);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const getSortedGoods = (orderBy, reverse = false) => {
+    const sorted = [...goodsFromServer];
```

Comment:
```suggestion
    const sorted = [...goodsFromServer];
```
looks like this is copy not sorted
use better naming